pkg/node/local_node.go

```diff
@@ -73,5 +75,12 @@ func ConfigureLocalNode() error {
 // NotifyLocalNodeUpdated Update local node information in the key-value
 // storage
 func NotifyLocalNodeUpdated() error {
```

Comment:
Not blocking, but you can remove the returned `error` because it is always `nil` with your change.